contracts/RiskManagerV1.sol

```diff
@@ -823,29 +687,17 @@ contract RiskManagerV1 is IRiskManagerV1, Auctioneer, Ownable {
 /* solhint-disable-next-line ordering */
 library RiskManagerV1Rewards {
     struct Storage {
-        // Fixed amount of COV tokens which should be given as reward for the
+        // Amount of COV tokens which should be given as reward for the
         // notifier reporting about the start of deposit liquidation process.
         uint256 liquidationNotifierRewardAmount;
```

Comment:
Let's take the opportunity and simplify the naming by removing the `Amount` part from field, parameter, and function names given that we no longer have to keep amount and percentage together:
```
liquidationNotifierRewardAmount -> liquidationNotifierReward
liquidatedNotifierRewardAmount -> liquidatedNotifierReward
```

Similarly for functions and parameters. For example:
```
beginLiquidatedNotifierRewardAmountUpdate -> beginLiquidatedNotifierRewardUpdate
finalizeLiquidatedNotifierRewardAmountUpdate -> finalizeLiquidatedNotifierRewardUpdate
```

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/146/commits/b57fb989c20a943dbe6d224196f2bbc50360fa9b.